library/src/test/java/com/github/gfx/android/orma/test/RxObservableTest.java

```diff
@@ -143,4 +164,32 @@ public void deleterObservable() throws Exception {
         assertThat(count, is(1));
         assertThat(db.selectFromBook().where("title = ?", "today").valueOrNull(), is(nullValue()));
     }
+
+    @Test
+    public void exceptionInObservable() throws Exception {
+        class AbortInMapException extends RuntimeException{}
+
+        TestSubscriber<String> testSubscriber = TestSubscriber.create();
+        final List<String> mapped = new ArrayList<>();
+
+        db.selectFromBook()
+                .executeAsObservable()
+                .map(new Func1<Book, String>() {
+                    @Override
+                    public String call(Book book) {
+                        mapped.add(book.title);
+                        if (book.title.equals("friday")) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Position literals first in String comparisons](https://www.codacy.com/app/gfuji/Android-Orma/file/2131367773/issues/source?bid=3128773&fileBranchId=3148396#l181)